compute_endpoint/globus_compute_endpoint/cli.py

```diff
@@ -364,6 +365,18 @@ def _upgrade_funcx_imports_in_config(name: str, force=False) -> str:
 def read_config(endpoint_dir: pathlib.Path) -> Config:
     endpoint_name = endpoint_dir.name
 
+    try:
+        import funcx_endpoint
+
+        if Version(funcx_endpoint.__version__) < Version("2.0.0"):
+            msg = (
+                "To avoid compatibility issues with Globus Compute, please uninstall "
```

Comment:
I think we should only suggest the uninstall.  They must be running the globus-compute-endpoint version to hit this, so there shouldn't be a reason to keep funcx-endpoint?

Reply:
after discussion, I'm ok with leaving it in, but probably should have a longer explanation in the upgrade document later.

Reply:
Per your initial comment, I've trimmed the exception message to only suggest uninstalling.